npx @backstage/create-app -> yarn install fails on macOS with `env: python: No such file or directory`

## Expected Behavior

After `npx @backstage/create-app` I have a working backstage installation.

## Actual Behavior

```
% npx @backstage/create-app 
? Enter a name for the app [required] backtest

Creating the app...

 Checking if the directory is available:
  checking      backtest ✔ 

 Creating a temporary app directory:
  creating      temporary directory ✔ 

 Preparing files:
  copying       .dockerignore ✔ 
  templating    .gitignore.hbs ✔ 
  templating    .eslintrc.js.hbs ✔ 
  copying       .prettierignore ✔ 
  copying       README.md ✔ 
  copying       app-config.local.yaml ✔ 
  copying       app-config.production.yaml ✔ 
  templating    app-config.yaml.hbs ✔ 
  templating    backstage.json.hbs ✔ 
  templating    catalog-info.yaml.hbs ✔ 
  copying       lerna.json ✔ 
  templating    package.json.hbs ✔ 
  copying       tsconfig.json ✔ 
  copying       README.md ✔ 
  copying       entities.yaml ✔ 
  copying       org.yaml ✔ 
  copying       template.yaml ✔ 
  copying       catalog-info.yaml ✔ 
  copying       index.js ✔ 
  copying       package.json ✔ 
  copying       README.md ✔ 
  templating    .eslintrc.js.hbs ✔ 
  copying       Dockerfile ✔ 
  copying       README.md ✔ 
  templating    package.json.hbs ✔ 
  copying       index.test.ts ✔ 
  copying       index.ts ✔ 
  copying       types.ts ✔ 
  copying       app.ts ✔ 
  copying       catalog.ts ✔ 
  copying       auth.ts ✔ 
  copying       proxy.ts ✔ 
  copying       scaffolder.ts ✔ 
  templating    search.ts.hbs ✔ 
  copying       techdocs.ts ✔ 
  templating    .eslintrc.js.hbs ✔ 
  copying       cypress.json ✔ 
  templating    package.json.hbs ✔ 
  copying       android-chrome-192x192.png ✔ 
  copying       favicon-16x16.png ✔ 
  copying       apple-touch-icon.png ✔ 
  copying       favicon-32x32.png ✔ 
  copying       favicon.ico ✔ 
  copying       index.html ✔ 
  copying       manifest.json ✔ 
  copying       robots.txt ✔ 
  copying       safari-pinned-tab.svg ✔ 
  copying       .eslintrc.json ✔ 
  copying       app.js ✔ 
  copying       App.test.tsx ✔ 
  copying       App.tsx ✔ 
  copying       apis.ts ✔ 
  copying       index.tsx ✔ 
  copying       setupTests.ts ✔ 
  copying       SearchPage.tsx ✔ 
  copying       LogoFull.tsx ✔ 
  copying       LogoIcon.tsx ✔ 
  copying       Root.tsx ✔ 
  copying       index.ts ✔ 
  copying       EntityPage.tsx ✔ 

 Moving to final location:
  moving        backtest ✔ 

 Building the app:
  executing     yarn install ◟ warning @backstage/cli > node-libs-browser > url > querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
warning lerna > @lerna/bootstrap > read-package-tree@5.3.1: The functionality that this package provided is now in @npmcli/arborist
warning lerna > @lerna/version > temp-write > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
warning lerna > @lerna/create > pacote > @npmcli/run-script > node-gyp > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning lerna > @lerna/bootstrap > @lerna/run-lifecycle > npm-lifecycle > node-gyp > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning lerna > @lerna/create > pacote > @npmcli/run-script > node-gyp > request > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
warning lerna > @lerna/create > pacote > @npmcli/run-script > node-gyp > request > har-validator@5.1.5: this library is no longer supported
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-api-docs > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/core-components > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/integration-react > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-catalog > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-catalog-import > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-catalog-react > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-catalog-graph > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-org > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-github-actions > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-scaffolder > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-tech-radar > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-techdocs-module-addons-contrib > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-user-settings > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-techdocs-react > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-search > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-techdocs > @material-ui/lab@4.0.0-alpha.57: You can now upgrade to @mui/lab. See the guide: https://mui.com/guides/migration-v4/
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/core-components > @material-table/core > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > backend > @backstage/backend-tasks > cron@1.8.3: Breaking dependency change (moved from moment to luxon). Please move to 2.0.0.
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > backend > @backstage/backend-common > aws-sdk > querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > backend > @backstage/backend-common > aws-sdk > uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > backend > @backstage/backend-common > aws-sdk > url > querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @testing-library/jest-dom > css > source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > backend > @backstage/plugin-techdocs-backend > @backstage/plugin-techdocs-node > @trendyol-js/openstack-swift-sdk > axios-cached-dns-resolve > babel-polyfill > core-js@2.6.12: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
warning workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > backend > @backstage/plugin-techdocs-backend > @backstage/plugin-techdocs-node > @trendyol-js/openstack-swift-sdk > axios-cached-dns-resolve > babel-polyfill > babel-runtime > core-js@2.6.12: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
warning Pattern ["app@link:packages/app"] is trying to unpack in the same destination "/Users/ii/Library/Caches/Yarn/v6/npm-app-0.0.0/node_modules/app" as pattern ["app@0.0.0","app@0.0.0"]. This could result in non-deterministic behavior, skipping.
warning "@backstage/cli > @hot-loader/react-dom-v16@16.14.0" has unmet peer dependency "react@^16.14.0".
warning "@backstage/cli > @hot-loader/react-dom-v17@17.0.2" has unmet peer dependency "react@17.0.2".
warning "@backstage/cli > @svgr/plugin-jsx@6.2.1" has unmet peer dependency "@svgr/core@^6.0.0".
warning "@backstage/cli > @svgr/plugin-svgo@6.2.0" has unmet peer dependency "@svgr/core@^6.0.0".
warning "@backstage/cli > react-hot-loader@4.13.0" has unmet peer dependency "react@^15.0.0 || ^16.0.0 || ^17.0.0 ".
warning "@backstage/cli > react-hot-loader@4.13.0" has unmet peer dependency "react-dom@^15.0.0 || ^16.0.0 || ^17.0.0 ".
warning "workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > eslint-plugin-cypress@2.12.1" has unmet peer dependency "eslint@>= 3.2.1".
warning "workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @testing-library/user-event@12.8.3" has unmet peer dependency "@testing-library/dom@>=7.21.4".
warning "workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/test-utils > @testing-library/user-event@14.2.0" has unmet peer dependency "@testing-library/dom@>=7.21.4".
warning "workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/core-components > @material-table/core@3.2.5" has unmet peer dependency "@date-io/core@^1.3.13".
warning "workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-scaffolder > @uiw/react-codemirror@4.7.0" has unmet peer dependency "@babel/runtime@>=7.11.0".
warning "workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/plugin-api-docs > graphiql > codemirror-graphql@1.3.0" has unmet peer dependency "@codemirror/language@^0.20.0".
warning "workspace-aggregator-6e230ef6-c9d7-4834-91f2-306a5b1b320e > app > @backstage/core-components > @material-table/core > @material-ui/pickers@3.3.10" has unmet peer dependency "@date-io/core@^1.3.6".
warning Error running install script for optional dependency: "/Users/ii/src/backtest/node_modules/cpu-features: Command failed.
Exit code: 1
Command: node buildcheck.js > buildcheck.gypi && node-gyp rebuild
Arguments: 
Directory: /Users/ii/src/backtest/node_modules/cpu-features
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@16.15.0 | darwin | arm64
gyp info find Python using Python version 3.9.12 found at \"/opt/homebrew/opt/python@3.9/bin/python3.9\"
(node:30817) [DEP0150] DeprecationWarning: Setting process.config is deprecated. In the future the property will be read-only.
(Use `node --trace-deprecation ...` to show where the warning was created)
gyp info spawn /opt/homebrew/opt/python@3.9/bin/python3.9
gyp info spawn args [
gyp info spawn args   '/Users/ii/src/backtest/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ii/src/backtest/node_modules/cpu-features/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ii/src/backtest/node_modules/@npmcli/run-script/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ii/Library/Caches/node-gyp/16.15.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/ii/Library/Caches/node-gyp/16.15.0',
gyp info spawn args   '-Dnode_gyp_dir=/Users/ii/src/backtest/node_modules/@npmcli/run-script/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/ii/Library/Caches/node-gyp/16.15.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/ii/src/backtest/node_modules/cpu-features',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_aarch64_linux_or_android.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_arm_linux_or_android.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_mips_linux_or_android.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_ppc_linux.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_x86_freebsd.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_x86_linux_or_android.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_x86_macos.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/impl_x86_windows.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/filesystem.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/stack_line_reader.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/string_view.o
  CC(target) Release/obj.target/cpu_features/deps/cpu_features/src/hwcaps.o
  LIBTOOL-STATIC Release/cpu_features.a
env: python: No such file or directory
make: *** [Release/cpu_features.a] Error 127
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/ii/src/backtest/node_modules/@npmcli/run-script/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (node:events:527:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
gyp ERR! System Darwin 21.5.0
gyp ERR! command \"/opt/homebrew/Cellar/node@16/16.15.0/bin/node\" \"/Users/ii/src/backtest/node_modules/.bin/node-gyp\" \"rebuild\"
gyp ERR! cwd /Users/ii/src/backtest/node_modules/cpu-features
gyp ERR! node -v v16.15.0
gyp ERR! node-gyp -v v7.1.2
gyp ERR! not ok"
error /Users/ii/src/backtest/node_modules/better-sqlite3: Command failed.
Exit code: 1
Command: prebuild-install || npm run build-release
Arguments: 
Directory: /Users/ii/src/backtest/node_modules/better-sqlite3
Output:
prebuild-install WARN install No prebuilt binaries found (target=16.15.0 runtime=node arch=arm64 libc= platform=darwin)

> better-sqlite3@7.5.3 build-release
> node-gyp rebuild --release

gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@16.15.0 | darwin | arm64
gyp info find Python using Python version 3.9.12 found at "/opt/homebrew/opt/python@3.9/bin/python3.9"
(node:31128) [DEP0150] DeprecationWarning: Setting process.config is deprecated. In the future the property will be read-only.
(Use `node --trace-deprecation ...` to show where the warning was created)
gyp info spawn /opt/homebrew/opt/python@3.9/bin/python3.9
gyp info spawn args [
gyp info spawn args   '/Users/ii/src/backtest/node_modules/@npmcli/run-script/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ii/src/backtest/node_modules/better-sqlite3/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ii/src/backtest/node_modules/@npmcli/run-script/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/ii/Library/Caches/node-gyp/16.15.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/ii/Library/Caches/node-gyp/16.15.0',
gyp info spawn args   '-Dnode_gyp_dir=/Users/ii/src/backtest/node_modules/@npmcli/run-script/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/ii/Library/Caches/node-gyp/16.15.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/ii/src/backtest/node_modules/better-sqlite3',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  TOUCH ba23eeee118cd63e16015df367567cb043fed872.intermediate
  ACTION deps_sqlite3_gyp_locate_sqlite3_target_copy_builtin_sqlite3 ba23eeee118cd63e16015df367567cb043fed872.intermediate
  TOUCH Release/obj.target/deps/locate_sqlite3.stamp
  CC(target) Release/obj.target/sqlite3/gen/sqlite3/sqlite3.o
  LIBTOOL-STATIC Release/sqlite3.a
env: python: No such file or directory
make: *** [Release/sqlite3.a] Error 127
rm ba23eeee118cd63e16015df367567cb043fed872.intermediate
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/ii/src/backtest/node_modules/@npmcli/run-script/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (node:events:527:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
gyp ERR! System Darwin 21.5.0
gyp ERR! command "/opt/homebrew/Cellar/node@16/16.15.0/bin/node" "/Users/ii/src/backtest/node_modules/.bin/node-gyp" "rebuild" "--release"
gyp ERR! cwd /Users/ii/src/backtest/node_modules/better-sqlite3
gyp ERR! node -v v16.15.0
gyp ERR! node-gyp -v v7.1.2
gyp ERR! not ok
yarn install v1.22.19
info No lockfile found.
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
[4/5] Linking dependencies...
[5/5] Building fresh packages...
info This module is OPTIONAL, you can safely ignore this error
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
  executing     yarn install ✖ 

Error: Could not execute command yarn install

It seems that something went wrong when creating the app 🤔

🔥  Failed to create app!
```

## Steps to Reproduce

1. Get an M1 Mac with macOS Monterey 12.4
2. Install Homebrew
3. Install Node with `brew install node@16` and link it with `brew link node@16`
4. Run `npx @backstage/create-app`

## Context

I can install various versions of Python with Homebrew. I have version 3.9 and 3.10 installed at the moment. Neither is automatically linked as `python` by Homebrew. macOS does not ship with Python anymore.

## Your Environment

- macOS Monterey 12.4
- Apple MacBook Air M1

--- a/.changeset/large-kangaroos-poke.md
+++ b/.changeset/large-kangaroos-poke.md
@@ -1,0 +1,13 @@
+---
+'@backstage/create-app': patch
+---
+
+Added an explicit `node-gyp` dependency to the root `package.json`. This is to work around a bug in older versions of `node-gyp` that causes Python execution to fail on macOS.
+
+You can add this workaround to your existing project by adding `node-gyp` as a `devDependency` in your root `package.json` file:
+
+```diff
+   "devDependencies": {
++    "node-gyp": "^9.0.0"
+   },
+```

--- a/packages/create-app/templates/default-app/package.json.hbs
+++ b/packages/create-app/templates/default-app/package.json.hbs
@@ -35,7 +35,8 @@
     "concurrently": "^6.0.0",
     "lerna": "^4.0.0",
     "prettier": "^2.3.2",
-    "typescript": "~4.6.4"
+    "typescript": "~4.6.4",
+    "node-gyp": "^9.0.0"
   },
   "resolutions": {
     "@types/react": "^17",

